### PR TITLE
Add contact section

### PR DESCRIPTION
**Pull request by Richie78321**
Adds a basic contact section at the bottom of the page using font awesome icons.

Also -- is there a point where a PR is too small? Lmk

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -11,6 +11,9 @@
 
     <link rel="stylesheet" href="./css/master.css">
 
+    <!-- FontAwesome Icons -->
+    <script src="https://kit.fontawesome.com/64201e6bf1.js" crossorigin="anonymous"></script>
```
Comment (billyeh) [inline portfolio/src/main/webapp/index.html:15]: is crossorigin required? Just wondering, I think anonymous is the default right but interested to know if there's a reason to specify it here.

Comment (Richie78321) [reply portfolio/src/main/webapp/index.html:15]: Admittedly I'm not sure. This was the default copy-paste for the FA CDN. Looking [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin) it seems like you're right about the value being the default but if the attribute isn't defined at all it doesn't use CORS whatsoever:
> By default (that is, when the attribute is not specified), CORS is not used at all. The "anonymous" keyword means that there will be no exchange of user credentials via cookies, client-side SSL certificates or HTTP authentication as described in the Terminology section of the CORS specification, unless it is in the same origin.

Comment (Richie78321) [reply portfolio/src/main/webapp/index.html:15]: At least with Bootstrap, which also includes crossorigin anonymous, when you remove it they complain about not being able to verify using CORS, and they don't deliver the content. Interesting. Thanks for the brief rabbithole lol.

Comment (billyeh) [reply portfolio/src/main/webapp/index.html:15]: I was looking at the same site, sorry yeah anonymous is not the same as not using CORS. I see so some servers nowadays I suppose expect CORS, and some don't.

**Pull request merged**